completion/git-machete.completion.zsh

```diff
@@ -25,7 +25,7 @@ _git-machete() {
           ;;
         (advance|delete-unmanaged)
           _arguments \
-            '(-y --yes)'{-y,--yes}'[Do not ask for confirmation]' \
+            '(-y --yes)'{-y,--yes}'[Do not ask for confirmation when deleting unmanaged branches]' \
```

Comment:
Whoops but this case applies to both `advance` and `delete-unmanaged`, while the new version is only relevant to `delete-unmanaged`... pls split the cases